# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/gtest/rtpsources_unittests.cpp
# Commit: d838dc7d2bb8
# Full Hash: d838dc7d2bb89ab5c9c2e8e93895db1c9310560f
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 4
# Description:
#   Bug 1654112 - Update include paths for moved upstream code. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D130059
# ==============================================================================

diff -r 587c40771588 -r d838dc7d2bb8 media/webrtc/signaling/gtest/rtpsources_unittests.cpp
--- a/media/webrtc/signaling/gtest/rtpsources_unittests.cpp	Fri Oct 23 13:49:51 2020 -0400
+++ b/media/webrtc/signaling/gtest/rtpsources_unittests.cpp	Fri Oct 23 14:02:54 2020 -0400
@@ -1,6 +1,6 @@
 #include <RtpSourceObserver.h>
 #include "RTCStatsReport.h"
-#include "webrtc/modules/include/module_common_types.h"
+#include "modules/include/module_common_types.h"
 #define GTEST_HAS_RTTI 0
 #include "gtest/gtest.h"
 